config/config.go

```diff
@@ -19,10 +19,12 @@ const (
 
 var version = "should-be-overridden-by-setDefaults"
 
-var Lookup func(string) ([]net.IP, error)
-var Dial func(string, string, time.Duration) (net.Conn, error)
-var Checkpoints []chaincfg.Checkpoint
-var TimeSource MedianTimeSource
+var (
+	Lookup      func(string) ([]net.IP, error)
+	Dial        func(string, string, time.Duration) (net.Conn, error)
+	Checkpoints []chaincfg.Checkpoint
+	TimeSource  MedianTimeSource
+)
```

Comment:
I'm just wondering, why these vars are in the `config.go` file 🤔 
Besides, if they are public each one should have a comment... or I'm too strict 🙈

Reply:
Those are used by the old p2p, so hopefully we will get rid of them when the new version will be finished

Reply:
@dorzepowski is right, I'm resolving this for now :)